frontend/views/components/Modal/ModalMixins.js

```diff
@@ -23,6 +32,22 @@ const modaMixins = {
     },
     unload () {
       sbp('okTurtles.events/emit', CLOSE_MODAL)
+    },
+    testFocus (e) {
```

Comment:
I think this needs a better name than "testFocus", which sounds like an unfinished work-in-progress